pubsub/integration_test.go

```diff
@@ -1329,8 +1329,8 @@ func TestIntegration_OrderedKeys_ResumePublish(t *testing.T) {
 		OrderingKey: orderingKey,
 	})
 	<-r.ready
-	if r.err != nil {
-		t.Fatalf("got error while publishing message: %v", r.err)
+	if r.Err != nil {
+		t.Fatalf("got error while publishing message: %v", r.Err)
```

Comment:
Have you tried running unit and system tests? Do they all pass, or at least those that pass in the master?

Reply:
The integration test fails because I don't have credentials for both master and branch. The rest of the tests passes.

Reply:
@IlyaFaer Can IT tests be run with the emulator in Go?

Reply:
fixed